app/wiring/AppComponents.scala

```diff
@@ -43,10 +43,15 @@ trait AppComponents extends PlayComponents with GzipFilterComponents {
     Cookies.authProvider(identityKeys).withDisplayNameProvider(Token.authProvider(identityKeys, "membership"))
 
   val contributionDataPerMode: Map[PaymentMode, ContributionData] = {
+    val postgresConfigId = "postgres"
+    val auroraConfigId = "aurora"
```

Comment:
this is only be used in one place - perhaps it can just be included in the string that is being interpolated using it?